src/lib/input/_input.scss

```diff
@@ -1,6 +1,9 @@
 @import '../core/style/variables';
 @import '../core/style/vendor-prefixes';
 
+
+@mixin mat-input-styles {
+
 // The Input element proper.
 .mat-input-element {
```

Comment:
Please indent the selectors within the `@mixin` by two spaces.

Reply:
Opted to just move the .scss file instead